docs/src/geometry/api/g/intersection/exists.html

```diff
@@ -0,0 +1,26 @@
+<pre class="docs-method-signature"><code>g.intersection.exists(shape1, shape2 [, shape1Options, shape2Options])</code></pre>
+<p>Returns <code>true</code> if two shapes intersect each other. The shape must be one of the following types:</p>
+<ul>
+    <li><a href="#g.Point">Point</a></li>
+    <li><a href="#g.Ellipse">Ellipse</a></li>
+    <li><a href="#g.Rect">Rect</a></li>
+    <li><a href="#g.Polyline">Polyline</a></li>
+    <li><a href="#g.Polygon">Polygon</a></li>
+    <li><a href="#g.Path">Path</a></li>
+</ul>
+
+<pre><code>
+g.intersection.exists(new g.Rect(0, 0, 10, 10), new g.Line({ x: 5, y: 5 }, { x: 30, y: 30 })); // true</code></pre>
+
+Using the the shape options is valid for <a href="#g.Path">Path</a> only. The path is internally convert into polylines with <a href="#g.Path.prototype.toPolylines">toPolylines()</a> method. The options control the conversion of the path.
```

Comment:
Looks good. The code comments are very clear.

"Using **the** shape options..." (1 'the').

"The path is internally **converted** into polylines with **the** ..."